packages/relay/src/lib/eth.ts

```diff
@@ -916,13 +916,17 @@ export class EthImpl implements Eth {
   async sendRawTransaction(transaction: string, requestId?: string): Promise<string | JsonRpcError> {
     const requestIdPrefix = formatRequestIdMessage(requestId);
     let interactingEntity = '';
-    this.logger.trace(`${requestIdPrefix} sendRawTransaction(transaction=${transaction})`);
+    let originatingAddress = '';
     try {
       const parsedTx = Precheck.parseTxIfNeeded(transaction);
       interactingEntity = parsedTx.to ? parsedTx.to.toString() : '';
+      originatingAddress = parsedTx.from ? parsedTx.from.toString() : '';
+      this.logger.trace(`${requestIdPrefix} sendRawTransaction(from=${originatingAddress}, to=${interactingEntity}, transaction=${transaction})`);
+
       const gasPrice = Number(await this.gasPrice(requestId));
       await this.precheck.sendRawTransactionCheck(parsedTx, gasPrice, requestId);
     } catch (e: any) {
+      this.logger.warn(`${requestIdPrefix} Error on precheck sendRawTransaction(from=${originatingAddress}, to=${interactingEntity}, transaction=${transaction})`);
```

Comment:
With this do you think the from and to could be noted in the pfecheck method itself.
That way we won't need an extra log.
If not this is fine.

Reply:
I think is doable, but that way, we won't log anything in case the parseTx fails...